docs/processing_information.md

```diff
@@ -53,6 +53,9 @@ This function more closely mimics the filtering performed in Cell Ranger than it
 We consider droplets with an FDR less than or equal to 0.01 to be cell-containing droplets. 
 Only cells that pass this FDR threshold are included in the filtered counts matrix.
 
+In the event that the cells do not meet the assumptions met in `DropletUtils::emptyDropsCellRanger` and are unable to be filtered using this method (e.g. have a lower than expected number of droplets), droplets were filtered based on total UMI count. 
```

Comment:
I wanted to simplify the structure of this a bit. Since the next sentence contains the filtering we do in the case of failure, I eliminated that clause.
```suggestion
For some libraries, `DropletUtils::emptyDropsCellRanger()` may fail due to low numbers of droplets with reads or other violations of its assumptions.
```